examples/mqtt/mqtt-shapes/README.md

```diff
@@ -350,8 +371,8 @@ menu items in the *RTI Shapes Demo* application:
 - "Publish/Circle..." and click 'OK' to use the default parameters.
 - "Publish/Triangle..." and click 'OK' to use the default parameters.
 
-> **NOTE**: The Windows `shapes_mqtt_publisher.bat` only publishes circles if
-> the topic is not specified.
+> **NOTE**: The Windows script `shapes_mqtt_publisher.bat` only publishes
+> circles if the topic is not specified.
```

Comment:
What does this mean?

Reply:
I see. I would suggest to make the wording a bit clearer, e.g something like:

>**NOTE**: By default, on Windows the demo will only publish topic "Circles" from MQTT. This is because script >`shapes_mqtt_publisher.bat` only publishes one topic at time, whereas the shell version publishes three topics at a time.

Please adjust/correct me if I misunderstood. I rephrased it because I think this is the "take away" for a user if I understand the issue correctly.

I now see though that there is another similar note a few lines above in the same section, so probably this note should be removed altogether...

Reply:
good point, I've removed the note